docs/ops/type/ConvertLike_1.md

```diff
@@ -2,37 +2,35 @@
 
 **Versioned name**: *ConvertLike-1*
 
-**Category**: type conversion
+**Category**: *Type conversion*
 
-**Short description**: Operation converts all elements of the 1st input tensor to a type of elements of 2nd input tensor.
+**Short description**: *ConvertLike* operation performs element-wise conversion on a given input tensor `data` to the element type of an additional input tensor `like`.
 
-**Attributes**:
+**Detailed description**
 
-    No attributes available.
+Conversion from one supported type to another supported type is always allowed. User must be aware of precision loss and value change caused by range difference between two types. For example, a 32-bit float *3.141592* may be round to a 32-bit int *3*. The result of unsupported conversions is undefined, e.g. convertion of negative signed integer value to any unsigned integer type.
 
-**Inputs**
+Ouput elements are represented as follows:
 
-* **1**: `data` - A tensor of type T1. **Required.**
-* **2**: `like` - A tensor of type T2. **Required.**
+    o[i] = Convert[destination_type=type(b)](a[i])
 
-**Outputs**
+where `a` and `b` correspond to `data` and `like` input tensors respectively.
 
-* **1**: The result of element-wise *"ConvertLike"* operation. A tensor of the same type with `like` tensor and the same shape with `data` tensor.
+**Attributes**: *ConvertLike* operation has no attributes.
 
-**Types**
+**Inputs**
 
-* *T1*: u8, u16, u32, u64, i8, i16, i32, i64, f16, f32, boolean, bf16
-* *T2*: u8, u16, u32, u64, i8, i16, i32, i64, f16, f32, boolean, bf16
+* **1**: `data` - A tensor of type *T1* and arbitrary shape. **Required.**
+* **2**: `like` - A tensor of type *T2* and arbitrary shape. **Required.**
 
-**Detailed description**
+**Outputs**
 
-Conversion from one supported type to another supported type is always allowed. User must be aware of precision loss and value change caused by range difference between two types. For example, a 32-bit float *3.141592* may be round to a 32-bit int *3*.
+* **1**: The result of element-wise *ConvertLike* operation applied to input tensor `data`. A tensor of type *T2* and the same shape as `data` input tensor.
 
-*a* - `data` input tensor, *b* - `like` input tensor.
+**Types**
 
-\f[
-o_{i} = Convert[destination_type=type(b)](a_{i})
-\f]
+* *T1*: `u8`, `u16`, `u32`, `u64`, `i8`, `i16`, `i32`, `i64`, `f16`, `f32`, `boolean`, `bf16`
+* *T2*: `u8`, `u16`, `u32`, `u64`, `i8`, `i16`, `i32`, `i64`, `f16`, `f32`, `boolean`, `bf16`
```

Comment:
Change to "any supported data type". Even if CPU plugin supports only these we don't want to limit our specification with exact types.